tests/components/wallbox/__init__.py

```diff
@@ -32,12 +33,13 @@
         {
             CONF_CHARGING_POWER_KEY: 0,
             CONF_MAX_AVAILABLE_POWER_KEY: 25.2,
-            CONF_CHARGING_SPEED_KEY: 0,
+            CONF_CHARGING_SPEED_KEY: "YYY",
```

Comment:
Why change it to string? Form what I can tell, it should still work, however since `precision=0` is set, a debug log entry should be emitted.
```
Cannot format ... Charging Speed
```

Reply:
Yes, I did this on purpose to trigger a formatting error in the sensor component (which is being caught by a try ... except in the code

Reply:
Is that a valid response from the API though?

Reply:
Just my opinion, I think it's safe to assume an API will return the datatypes it says it will. Testing it every time will just add unnecessary overhead. _The exception to this would of course be security reasons. I.e. don't pass an unescaped API response to a database._

If you really want to go that route, wouldn't you also need to check that the structure of `getChargerStatus`? The API response is just silently assigned to `data`. In the end, I don't believe those checks add much value. It's more important to guard against the expected errors, e.g. Connection and Authentication.

Reply:
OK, I see your point, it was useful in the early days when I wasn't so sure about the data structure. I propose to remove this (and the try... exception on the round) in a seperate PR (it is more sensor related than lock related). Agree?

Reply:
> I propose to remove this (and the try... exception on the round) in a seperate PR (it is more sensor related than lock related). Agree?

Sounds good to me. Let's just also revert that change here.

```suggestion
            CONF_CHARGING_SPEED_KEY: 0,
```